spec/models/course_spec.rb

```diff
@@ -695,38 +695,13 @@
     end
   end
 
-  context "UCAS level (deprecated)" do
-    context "with no ucas_subjects" do
-      subject { create(:course) }
-      its(:ucas_level) { should eq(:secondary) }
-    end
-
-    context "with primary ucas_subjects" do
-      subject { create(:course, level: "primary", ucas_subjects: [find_or_create(:ucas_subject, :primary)]) }
-      its(:ucas_level) { should eq(:primary) }
-      its(:gcse_subjects_required) { should eq(%w[maths english science]) }
-    end
-
-    context "with secondary ucas_subjects" do
```

Comment:
deleted too much?

given a `course`
when it has that `level`
then `gcse_subjects_required` is set correctly?